Cod3rsGrowth.Infra/Repositorio/DeletarJogosRepositorio.cs

```diff
@@ -0,0 +1,14 @@
+﻿using LinqToDB;
+
+namespace Cod3rsGrowth.Infra.Repositorio
+{
+    public class DeletarJogosRepositorio
+    {
+        public static void DeletarJogosAdicionadosEmTeste(DbCod3rsGrowth bancoDeDados)
+        {   
+            const int numeroMaximoDeJogos = 20;
+            bancoDeDados.Jogo
+                .Delete(jogo => jogo.Id > numeroMaximoDeJogos);
```

Comment:
essa e uma query perigosa , visto que vc conta que os primeiros 20 valores adicionado tem o id de 1 a 20 , as vezes pode ser que fica alocação na memoria e os ID adicionados pode ser acima de 20 e são objetos que vc conta para o teste, por exemplo se eu subir esse valor na minha maquina e os Id for dirente começar no 21, foque no objeto que vc que excluir por uma propiedade que vc sabe que vai ser valor unico que vc mesmo adicionou e não que o banco de dados criou para vc